.gitignore

```diff
@@ -15,3 +15,6 @@
 /libpeerconnection.log
 npm-debug.log
 testem.log
+
+# hoodie
```

Comment:
not sure what you mean? `# hoodie` is a comment to separate it from the sections above. What I added is ignoring the `.hoodie` folder, this is where the databases will be written into